recipes/nng/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} test_package.c)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::nng)
+set_property(TARGET ${PROJECT_NAME} PROPERTY C_STANDARD 99)
```

Comment:
This is the 2nd place you're explicitly setting the C standard.
Does the C standard influence the package?
```suggestion
```

Reply:
nng requires it and sets it in its own CMake script. I don't know if it's necessary to explicitly set it here though.